srcpkgs/thermald/files/thermald/log/run

```diff
@@ -0,0 +1 @@
+/usr/bin/vlogger
```

Comment:
```suggestion
#!/bin/sh
exec vlogger -t thermald
```

Reply:
I copied this from the docker package actually: https://github.com/void-linux/void-packages/blob/master/srcpkgs/moby/files/docker/log/run
So what is actually the best way? Is the docker package wrong?

Reply:
1. shebang ensures it's run correctly
2. exec makes it replace the "run the log service" process instead of running it in a subprocess, which is the proper way

Reply:
Should I open a PR to fix docker?

Reply:
it's actually a symlink, which github renders as a file containing one line :)

```
$ ls -l srcpkgs/moby/files/docker/log
lrwxrwxrwx 1 abby abby 16 Jun 20  2021 run -> /usr/bin/vlogger
```
so that's probably the proper solution

Reply:
Yes, I know and that's what this PR contrains too: a symlink to vlogger. Should I keep it then?

Reply:
oh, in that case, yes, keep it.